test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ValidateTests.cs

```diff
@@ -187,11 +191,133 @@ public async void GivenAValidateResourceToUSCoreProfile_WhenTheResourceProfileIs
                 {
                     hasSeenCodeInvalid = true;
                 }
+
+                break;
             }
 
             Assert.True(hasSeenCodeInvalid, "Resource has invalid code");
         }
 
+        // Validate-Code Post Tests Start
+        [SkippableTheory]
+        [InlineData("Parameter-ValueSet-Validate-Code-Correct")]
+        [InlineData("Parameter-CodeSystem-Validate-Code-Correct")] // May fail as CodeSystem resource changed STU3 -> R4 and the TS is on R4
+        public async void GivenValidParamaterInput_ValidateCodeReturnTrue(string fileName)
+        {
+            Skip.If(string.IsNullOrEmpty(_fhirServer.Configuration["FhirServer:Operations:Validate:ExternalTerminologyServer"]));
+            var fhirSource = Samples.GetJson(fileName);
+            Parameters resultParam = await _client.ValidateCodePOSTdAsync("ValueSet/$validate-code", fhirSource);
+            foreach (var paramComponenet in resultParam)
+            {
+                if (paramComponenet.Key == "result")
+                {
+                    Assert.Equal("true", paramComponenet.Value.ToString());
+                }
+
+                break;
```

Comment:
This break makes it so only the first paramComponent is checked. Is this intentional?

Reply:
I'm seeing this in all your foreaches in this file.

Reply:
Intentional as mentioned in previous resolved comment.